src/main/java/racingcar/CarCollection.java

```diff
@@ -1,32 +1,33 @@
 package racingcar;
 
+import java.sql.Array;
+import java.util.ArrayList;
+
 public class CarCollection {
-    private Car[] carList;
-    private int size;
+    private ArrayList<Car> carList;
```

Comment:
배열을 리스트로 바꾸신건 좋았지만, 참조타입은 최대한 추상화된 타입을 사용해주세요. 코드의 유연성을 확보할 수 있습니다
```suggestion
    private List<Car> carList;
```

지금은 ArrayList지만 나중엔 다른 구현체 List가 될 수도 있는데, 지금과 같은 참조타입은 항상 유지보수를 위해 변경해줘야 할 코드가 생기는 문제가 있죠